### PR TITLE
Add circleci support

### PR DESCRIPTION
**Pull request by nenadalm**

### CODE REVIEW

File: src/antq/ver/circle_ci_orb.clj [hunk 1]
```diff
@@ -0,0 +1,26 @@
+(ns antq.ver.circle-ci-orb
+  (:require
+   [clojure.java.io :as io]
+   [clojure.string :as str]
+   [clojure.data.json :as json]
+   [antq.ver :as ver]))
+
+(defn orb-id [ns name]
+  (-> (io/as-url (str "https://internal.circleci.com/api/v2/orbs?ns=" ns "&name=" name))
+      slurp
```
Comment (liquidz) [inline src/antq/ver/circle_ci_orb.clj:10]: Please add error handlings.
e.g. https://github.com/liquidz/antq/blob/79f42b50f99fa247fddfe8fd89cdc494b1b96dce/src/antq/ver/github_tag.clj#L59-L67

File: src/antq/ver/circle_ci_orb.clj [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+      (json/read-str :key-fn keyword)
+      :items
+      first
+      :id))
+
+(defn orb-versions [id]
+  (-> (io/as-url (str "https://internal.circleci.com/api/v2/orbs/" id))
+      slurp
```
Comment (liquidz) [inline src/antq/ver/circle_ci_orb.clj:18]: Please add error handlings here too.

**Review by liquidz — CHANGES_REQUESTED** [review]
@nenadalm Thanks for your contribution!
This is awesome work!

There are a few points I’d like you to revise, so could you check them?

**Comment by nenadalm** [comment]
Thanks. All remarks should be fixed now.

**Review by liquidz — APPROVED** [review]
@nenadalm LGTM :) Thanks!

**Comment by liquidz** [comment]
@nenadalm Just released v2.11.1250 :)

**Pull request merged**